### DISCUSSION TITLE
How to create a registration page (website)?

### DISCUSSION THREAD

**Discussion by Vergulhos** [azerothcore/azerothcore-wotlk#18236] [General]
I set up a registration page, it works when registering, but I can't get the "salt" and "verifier" part to work.
![Captura de tela 2024-01-27 233707](https://github.com/azerothcore/azerothcore-wotlk/assets/157935077/6d9d8f18-528d-4021-8fea-1acd3b6484d6)
![Captura de tela 2024-01-27 233824](https://github.com/azerothcore/azerothcore-wotlk/assets/157935077/8e1c96c9-9bb7-4fbe-a4fb-7bf2725b2f79)

He registers, but does not log in.

Which SALT and VERIFIER command do I use in PHP?

**Comment by danperor** [comment]
see this project :https://github.com/masterking32/WoWSimpleRegistration
it's php,too
